specs/composition/common/schemas/responses/runResponseComponents/CompositionsRunSection.yml

```
@@ -0,0 +1,59 @@
+# Schema for the compositions section in a composition run response.
+# {
+#   ...
+#   "compositions": {
+#     "run": [
+#       {
+#         "objectID": "comp1765458818347",
+#         "appliedRules": [
+#           {
+#             "objectID": "cr-1765458959657"
+#           }
+#         ]
+#       }
+#     ]
+#   }
+# }
+
+compositionBaseSearchResponse:
+  type: object
+  properties:
+    compositions:
+      $ref: '#/compositionsSearchResponse'
+
+compositionsSearchResponse:
```

Is it a breaking change to make this consistent with the others?
```suggestion
compositionSearchResponse:
```

---

I admit I kept all the existing names because otherwise it triggers breaking changes on the client side not visible before running the CI.
I am open to challenge names 1o1 as a followup of this PR 😅